internal/database/actions.go

```diff
@@ -48,20 +106,18 @@ func GetResultTestKeys(sess sqlbuilder.Database, resultID int64) (string, error)
 		tk   PerformanceTestKeys
 	)
 	for res.Next(&msmt) {
-		if msmt.TestName == "web_connectivity" {
-			break
-		}
 		// We only really care about performance keys.
 		// Note: since even in case of failure we still initialise an empty struct,
 		// it could be that these keys come out as initializes with the default
 		// values.
 		// XXX we may want to change this behaviour by adding `omitempty` to the
 		// struct definition.
-		if msmt.TestName == "ndt" || msmt.TestName == "dash" {
-			if err := json.Unmarshal([]byte(msmt.TestKeys), &tk); err != nil {
-				log.WithError(err).Error("failed to parse testKeys")
-				return "{}", err
-			}
+		if (msmt.TestName != "ndt" && msmt.TestName != "dash") {
```

Comment:
please remove the unnecessary parentheses

Reply:
Only here you don't like them eh? :P

Reply:
I mean, if you use goreportcard.com and include parentheses in this context, it will reduce your score and suggest to use `gofmt -s` to simplify the codebase.